README.md

```diff
@@ -1,24 +1,108 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Hello-Rails-Back-End
 
-Things you may want to cover:
+> This is an educational project to set up a "Hello world" project as two separated apps.
 
-* Ruby version
+Front-End: [Hello-Rails-Front-End](https://github.com/mavericks-db/hello-react-front-end)
```

Comment:
- [ ] Please check the code on the front end, I am getting an infinite loop:

https://user-images.githubusercontent.com/77038610/189905818-7b431460-c75f-40b5-853a-44aacb865de5.mp4

Reply:
Resolved in the front-end component.